src/components/Dialog/components/DialogHeader.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from 'react';
+import IconButton from 'components/IconButton';
+import DenyIcon from 'lib/icons/react/DenyIcon';
+import DialogContext from '../DialogContext';
+
+const DialogHeader = () => {
+  const { onDismiss, title } = useContext(DialogContext);
+  return (
+    <>
+      {(title || onDismiss) && (
+        <div className="bgColor--white border--bottom">
+          <div className="dialog-header padding--left padding--top padding--bottom">
+            <div className="type--head4">{title}</div>
```

Comment:
TODO: come back and ensure there's an empty string here when no title is passed.